src/store/modules/actions/staticData.js

```diff
@@ -88,7 +85,10 @@ export async function lazyGetNFTPurchaseInfo({ getters, dispatch }, classId) {
 
 export async function fetchNFTMetadata({ commit }, classId) {
   let metadata;
-  const chainMetadata = await getClassInfo(classId);
+  // const chainMetadata = await getClassInfo(classId);
```

Comment:
Should remove this?

Reply:
want to remind ppl this is a performance hack, maybe i add a comment